test.js

```diff
@@ -1,49 +1,72 @@
 'use strict';
 
 var MergeTrees = require('./');
+const { createBuilder, createTempDir } = require('broccoli-test-helper');
```

Comment:
Please split the test changes into a separate PR, I would like to land the class update separately

Reply:
Done #73